app/models/message.rb

```diff
@@ -62,17 +26,23 @@ def increase_unread(user)
     end
   end
 
-  def notification_type(user, person)
-    Notifications::PrivateMessage unless user.person == person
-  end
-
   def message
     @message ||= Diaspora::MessageRenderer.new text
   end
 
+  # @return [Array<Person>]
+  def subscribers
+    if author.local?
+      conversation.participants
+    else # for relaying, TODO: can be removed when messages are not relayed anymore
+      conversation.participants.select(&:remote?)
```

Comment:
Same again, is the whole collection loaded anyway? If not can we turn this into a DB query?